-------------------------------------------------------------------------------
Test set: model.UserTestCase
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.003 s <<< FAILURE! - in model.UserTestCase
test(model.UserTestCase)  Time elapsed: 2.003 s  <<< ERROR!
org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
	at model.UserTestCase.initTest(UserTestCase.java:21)
Caused by: org.hibernate.exception.JDBCConnectionException: Error calling Driver#connect
	at model.UserTestCase.initTest(UserTestCase.java:21)
Caused by: com.mysql.cj.jdbc.exceptions.CommunicationsException: 
Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at model.UserTestCase.initTest(UserTestCase.java:21)
Caused by: com.mysql.cj.exceptions.CJCommunicationsException: 
Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at model.UserTestCase.initTest(UserTestCase.java:21)
Caused by: java.net.ConnectException: Connection refused: connect
	at model.UserTestCase.initTest(UserTestCase.java:21)

